ext/search_kit/CRM/Search/Upgrader.php

```diff
@@ -121,8 +121,7 @@ public function upgrade_1003(): bool {
    */
   public function upgrade_1005(): bool {
     $this->ctx->log->info('Applying update 1005 - add acl_bypass column.');
-    $this->addTask('Add Cancel Button Setting to the Profile', 'addColumn',
-      'civicrm_search_display', 'acl_bypass', "tinyint DEFAULT 0 COMMENT 'Skip permission checks and ACLs when running this display.'");
+    $this->addColumn('civicrm_search_display', 'acl_bypass', "tinyint DEFAULT 0 COMMENT 'Skip permission checks and ACLs when running this display.'");
```

Comment:
Note this is copied from `upgrade_1001()` above. If @demeritcowboy's simpler version works there, it will work here too.